GitUI/CommandsDialogs/FormRebase.cs

```diff
@@ -381,17 +381,22 @@ private void btnChooseFromRevision_Click(object sender, EventArgs e)
             bool previousValueBranchFilterEnabled = AppSettings.BranchFilterEnabled;
             bool previousValueShowCurrentBranchOnly = AppSettings.ShowCurrentBranchOnly;
             bool previousValueShowReflogReferences = AppSettings.ShowReflogReferences;
+            bool previousValueShowStashes = AppSettings.ShowStashes;
 
             try
             {
-                using FormChooseCommit chooseForm = new(UICommands, txtFrom.Text, showCurrentBranchOnly: true);
+                AppSettings.ShowStashes = false;
+                ObjectId firstParent = UICommands.GitModule.RevParse(Currentbranch.Text + "~");
```

Comment:
Side note: This should be queried from RevGrid to avoid extra Git commands. It is Git access that slows down the app...

Reply:
I disagree in this case.
In this control we don't have access to the revision grid and I had a look, it will add a lot of logic to be able to pass the needed information as the form browse is called in multiple case (rebase, interactive, rebase onto, ....). So more than 4 or 6 different cases to handle.

And for the performance, doing a revparse is very quick (around 40ms) which is much less than requesting the logs to display the revision grid and opening the form. And also the goal of this feature is to make the use more convenient when you want to rebase onto only one commit and in this case, you probably gain 2 or 3s. So the net gain should be good 😉